src/transaction-runner.ts

```diff
@@ -81,7 +82,7 @@ interface ErrorCallback {
  *
  * @param {Error} [err] The last known retryable Error.
  */
-export class DeadlineError extends Error implements ServiceError {
+export class DeadlineError extends Error implements RowCountsServiceError {
```

Comment:
I don't think we should be extending `RowCountsServiceError` here

Reply:
I agree that it doesn't seem correct, and maybe it's just my IDE throwing false errors, but `ServiceError` doesn't seem to get extended properly because of its `rowCounts` method. I'll play around with it 👍

Reply:
@callmehiphop just wanted to show you [this](https://imgur.com/a/Vc8tmf0) real quick

Reply:
How come `BatchUpdateError` doesn't just extend `ServiceError`? It extends what appears to be a duplicate interface and this error is probably because it has the same properties with different types.

Reply:
I think I found the problem - something was up with `node_modules/grpc/index.d.ts` so I had to re-`npm i`, you aren't going crazy 😄

Reply:
Lol, well lets not jump to any conclusions about that yet